package.json

```diff
@@ -1,25 +1,19 @@
 {
-  "name": "@nicky-lenaers/ngx-scroll-to-demo",
-  "description": "Demo Application for @nicky-lenaers/ngx-scroll-to package.",
+  "name": "@newforma/ngx-scroll-to-demo",
+  "description": "Forked demo application and package from @nicky-lenaers/ngx-scroll-to.",
   "version": "11.0.0",
   "license": "MIT",
   "scripts": {
     "ng": "ng",
     "start": "ng serve ngx-scroll-to-demo",
-    "build": "ng build --prod",
+    "build": "ng build --prod && npm run post-build",
+    "post-build": "copyfiles -u 2 'dist/ngx-scroll-to/**/*' .",
```

Comment:
You don't need use `copyfiles`. You can update `tsconfig.json` output settings.

Reply:
I'm not sure how to set it. When I edit the `outDir` to: `/dist/ngx-scroll-to` the package output is still whole project which is not we expect.

Reply:
sorry, I forger that it is angular. It will be easier to keep `copyfiles`